object-construction-checker/tests/socket/BindChannel.java

```diff
@@ -15,7 +15,7 @@ static void test(InetSocketAddress addr, boolean b) {
             // it's okay to keep issuing this error; see the second method in this class
             // for an example of how to rewrite the code to avoid the parse error.
             //
-            // :: error: required.method.not.called
+            // TODO we should report a warning here
```

Comment:
This is okay for now, because we report the error below. I'm investigating this.